docs/development.rst

```diff
@@ -133,6 +133,137 @@ To get an idea about the size distribution of our react codebase you can run::
 
 This will open an interactively explorable report in your browser.
 
+Add a React page
+----------------
+
+First we need to set up a few things in Django:
+
+#. Create a new Jinja template that our view will render (we'll use ``example.html``). Paste the following code in ``example.html`` to get started::
```

Comment:
Perhaps be a bit more specific as to where a file such as this would be added. Say for example we are adding a new page to the user management "app", where would `example.html` go?

Reply:
Great idea-- I think being specific will help with clarity. I'm going to use `kuma/dashboards` as an example.